lib/checkbufferoverrun.cpp

```diff
@@ -383,7 +383,7 @@ static bool checkMinSizes(const std::vector<Library::ArgumentChecks::MinSize> &m
                         parameters.push_back(nullptr);
                 }
                 const MathLib::biguint len = CheckBufferOverrun::countSprintfLength(formatstr, parameters);
-                if (len > arraySize + 2U)
+                if (len > arraySize + 2ULL)
```

Comment:
This is just to address compilation warning. Actually, simple "2" looks better and also fixes warning.